src/character.cpp

```diff
@@ -9862,7 +9941,7 @@ ret_val<bool> Character::can_wield( const item &it ) const
     if( has_effect( effect_incorporeal ) ) {
         return ret_val<bool>::make_failure( _( "You can't wield anything while incorporeal." ) );
     }
-    if( get_working_arm_count() <= 0 ) {
+    if( !has_min_manipulators() ) {
         return ret_val<bool>::make_failure(
                    _( "You need at least one arm to even consider wielding something." ) );
```

Comment:
```suggestion
                   _( "You need at least one arm available to even consider wielding something." ) );
```
I'm also bothered by it specifying "arm".